pytest.ini

```diff
@@ -10,4 +10,3 @@
 filterwarnings =
     error
     ignore:decodestring:DeprecationWarning
-    ignore:.*collections\.abc:DeprecationWarning
```

Comment:
Nit : The text at https://github.com/certbot/certbot/pull/7707/files#diff-513c9b6861fc981fb2c708db477f542cR7 could also be removed.

Reply:
Thanks, removed!